VerilogCodingStyle.md

```diff
@@ -2163,10 +2186,11 @@ always_comb begin
 end
 ```
 
-##### Wildcards in case items
+#### Wildcards in case items
 
-Use `case` instead of `casez` whenever wildcard operator behavior is not
-required. When wildcard behavior is needed, use `casez`.
+Use `case` instead of `case inside` nor `casez` whenever wildcard operator
```

Comment:
I believe this is the right interpretation, but I'm unable to scan our repository at the moment (machine issues) to confirm that we're following this guidance. Also looking to get confirmation from @eunchan or @imphil or others.

Reply:
@sjgitty we do have one case in the Ibex tracer where we use `casez`, but that's easy to fix.